### PR TITLE
hack/verify-staging-imports.sh: check that plugins are not imported b…

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sttts** [comment]
@k8s-bot node e2e test this

**Comment by sttts** [comment]
@k8s-bot cvm gke e2e test this
@k8s-bot gci gke e2e test this
@k8s-bot gci gce e2e test this
@k8s-bot node e2e test this

**Comment by sttts** [comment]
@k8s-bot cvm gke e2e test this
@k8s-bot gci gke e2e test this

File: hack/verify-staging-imports.sh [hunk 1]
```diff
@@ -77,4 +77,11 @@ if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
 	exit 1
 fi
 
+IN_CLUSTER_EXAMPLE=vendor/k8s.io/client-go/examples/in-cluster
+test -d "${IN_CLUSTER_EXAMPLE}" # make sure example is still there
```
Comment (caesarxuchao) [inline hack/verify-staging-imports.sh]: Can we test the tpr example instead? That uses more client-go features and thus has a better coverage.

Comment (sttts) [inline hack/verify-staging-imports.sh]: I think the tpr example imports the plugins (for some reason) ;-)

**Comment by sttts** [comment]
@k8s-bot cvm gke e2e test this
@k8s-bot gci gke e2e test this

**Comment by sttts** [comment]
@smarterclayton approved?

File: hack/verify-staging-imports.sh (hunk 1 above)
Comment (caesarxuchao) [inline hack/verify-staging-imports.sh]: Let's just comment out that import (but keep the line there to show people how to use the plugin if needed)

Comment (sttts) [inline hack/verify-staging-imports.sh]: done. Testing all three examples now.

**Comment by caesarxuchao** [comment]
```
I0307 22:45:38.388] Run ./hack/update-bazel.sh
I0307 22:45:38.389] FAILED   hack/make-rules/../../hack/verify-bazel.sh	6s
```

**Comment by sttts** [comment]
@k8s-bot cvm gke e2e test this
@k8s-bot kubemark e2e test this
@k8s-bot kops aws e2e test this
@k8s-bot non-cri e2e test this

**Comment by sttts** [comment]
@k8s-bot unit test this
@k8s-bot cvm gce e2e test this

**Pull request merged**